config/webhook/ingress.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: ingress-mctc
+  name: ingress
```

Comment:
Would webhook be a more descriptive name? We know it's an ingress from the kind of resource.

Reply:
Agreed. Just pushed a commit with the change